src/testTools.ts

```diff
@@ -0,0 +1,114 @@
+import request from "supertest";
+
+import { Provider, Role } from "./services/auth/auth-models.js";
+
+export const TESTER = {
+    id: "bob-the-tester101010101011",
+    email: "bob-the-tester@hackillinois.org",
+    name: "Bob Tester",
+};
+
+const AUTH_ROLE_TO_ROLES: Record<Role, Role[]> = {
+    [Role.APPLICANT]: [Role.APPLICANT],
+    [Role.USER]: [Role.USER],
+    [Role.ATTENDEE]: [Role.ATTENDEE],
+    [Role.STAFF]: [Role.ATTENDEE, Role.STAFF],
+    [Role.ADMIN]: [Role.ATTENDEE, Role.STAFF, Role.ADMIN],
+
+    [Role.MENTOR]: [Role.MENTOR],
+    [Role.SPONSOR]: [Role.SPONSOR],
+    [Role.BLOBSTORE]: [Role.BLOBSTORE],
+};
```

Comment:
we might need to make a few changes to this, can you add roles as such (note that we can never remove them):

<img width="1042" alt="image" src="https://github.com/HackIllinois/adonix/assets/49540156/506ab29c-3b65-4f48-921e-d3d715c7c7d8">

Reply:
Resolved by 17bacb6